src/test/java/io/codeka/gaia/e2e/SeleniumIT.java

```diff
@@ -40,7 +40,11 @@
     private static MongoContainer mongoContainer = new MongoContainer()
             .withScript("src/test/resources/db/00_team.js")
             .withScript("src/test/resources/db/10_user.js")
-            .withScript("src/test/resources/db/20_module.js");
+            .withScript("src/test/resources/db/20_module.js")
+            .withScript("src/test/resources/db/30_stack.js")
+            .withScript("src/test/resources/db/41_job.js")
```

Comment:
41 for job and 40 for step for the order in the database scripts ? Steps should be after jobs I think considering the model.

Reply:
Yes you're right. It's a mistake !